canonical-web-and-design/jaas-dashboard
# Replace node sass with sass

## Done

`node-sass` is now deprecated and [cannot be updated anyway](https://github.com/canonical-web-and-design/jaas-dashboard/pull/741).

Dart sass aka `sass` is now the only horse in town.

## QA

- Pull code
- Run `./run clean && ./run serve`
- Open http://0.0.0.0:8036/
- Verify everything looks as expected
